backend/development.ini.sample

```diff
@@ -761,6 +761,26 @@ caldav.radicale.headers.Access-Control-Expose-Headers = Etag
 ; collaborative_document_edition.file_template_dir = %(here)s/tracim_backend/templates/open_documents
 
 
+####
+# Translation service
+####
+
+# This feature permit to translate some content/comments by using external services
```

Comment:
Not sure about this wording `content/comments` because you says `translation of Note and Comments through` in settings.md

I think its better to write `Note/Comments` or to precise that content is not file content.

Reply:
```suggestion
# This feature permit to translate notes and comments by using external services
```

Reply:
fixed with a different string.